saleor/settings.py

```diff
@@ -138,6 +139,16 @@ def get_bool_from_env(name, default_value):
 
 ENABLE_SSL = get_bool_from_env("ENABLE_SSL", False)
 
+USER_EMAIL_HOST_USER: str = user_email_config.get("EMAIL_HOST_USER", "")
+USER_EMAIL_HOST_PASSWORD: str = user_email_config.get("EMAIL_HOST_PASSWORD", "")
+USER_EMAIL_HOST: str = user_email_config.get("EMAIL_HOST", "")
+USER_EMAIL_PORT = user_email_config.get("EMAIL_PORT", "")
+if USER_EMAIL_PORT:
+    USER_EMAIL_PORT = str(USER_EMAIL_PORT)
```

Comment:
If you're making casting here, then probably would be good to also validate if the port is `int`.

Reply:
Such validation is done by django-email-url when parsing email url.
This here is just handling case where `EMAIL_PORT` value would be `None` and prevents it casting that into string.